platform/src/main/java/org/stellar/anchor/platform/paymentobserver/StellarPaymentObserver.java

```diff
@@ -120,4 +121,34 @@ public StellarPaymentObserver build() {
       return new StellarPaymentObserver(horizonServer, accounts, observers, pageTokenStore);
     }
   }
+
+  public static void main(String[] args) throws IOException, InterruptedException {
+    KeyPair account1 =
+        KeyPair.fromSecretSeed("SCBYEX2YH7BH5WVKVR22RW2M3QQR3P2P3NLWIVNNNEZHJ3KZ52E2QKZN");
+    KeyPair account2 =
+        KeyPair.fromSecretSeed("SDPJLASIYSGX7ZKOJZIGJGYGC5F6XKHTPEA7NR2Y6YKKCHIBR2GAPCEZ");
+
+    StellarPaymentObserver watcher =
+        builder()
+            .horizonServer("https://horizon-testnet.stellar.org")
+            .addAccount(account1.getAccountId())
+            .addAccount(account2.getAccountId())
+            .addObserver(
+                new PaymentListener() {
+                  @Override
+                  public void onReceived(PaymentOperationResponse payment) {
+                    System.out.println("Received:" + new Gson().toJson(payment));
+                  }
+
+                  @Override
+                  public void onSent(PaymentOperationResponse payment) {
+                    System.out.println("Sent:" + new Gson().toJson(payment));
+                  }
+                })
+            .build();
+
+    watcher.start();
+    Thread.sleep(300000);
+    watcher.shutdown();
+  }
```

Comment:
Did you mean to commit this? It's just for tests, right?

Just checking, I'm fine with it being in the code, so we can test it easily.

Reply:
😄